moto/awslambda/models.py

```diff
@@ -830,6 +829,7 @@ def _invoke_lambda(self, event: Optional[str] = None) -> Tuple[str, bool, str]:
         except docker.errors.DockerException as e:
             # Docker itself is probably not running - there will be no Lambda-logs to handle
             msg = f"error running docker: {e}"
+            logger.error(msg)
```

Comment:
Looks like this change is no longer necessary

Reply:
I would keep the logging of Docker errors, as in case of fail situations this would be very useful to have. Eg a new testcase is added, where the `@requires_docker` is not used. In case of a failing test, it is not always obvious when looking at the pytest outcome why this was caused. Being able to use eg '-o log_cli=true' would provide help here.